### DISCUSSION TITLE
Bots on BG

### DISCUSSION THREAD

**Discussion by artDeil4ek** [trickerer/Trinity-Bots#466] [General]
Bots on BG are all for the Horde.
Because of this, BG closes due to insufficient number of players in 
![WoWScrnShot_052523_204751](https://github.com/trickerer/Trinity-Bots/assets/62905878/47509b6b-9f85-4f47-98d3-2023b9e9abd4)
4 minutes
![WoWScrnShot_052523_154647](https://github.com/trickerer/Trinity-Bots/assets/62905878/af7a7bd8-e1d8-45bb-b3cb-a74bafb17a85)

**Comment by trickerer** [comment]
This doesn't happen on clean core+bots. You have custom changes besides bots so I can't help you.

As for all bots being listed as horde players - this just a client limitation. Core can only send guids to the client in BG score packets, client always assumes they're player guids, can't find those players and lists those entries as default team which is horde.
Bots are added to teams based on their race default faction.

**Reply by artDeil4ek** [reply]
Thank you! I'm using a server with bots that you gave in your repository. Custom changes only in DB ->Battleground_template ->WSG & Arathi **MaxPlayersPerTeam** and only that.
(https://github.com/trickerer/TrinityCore-3.3.5-with-NPCBots))
Or do I need to take a clean TrinityCore and patch it myself?

**Reply by trickerer** [reply]
No, you don't need to re-build anything. Just test it with standard `MaxPlayersPerTeam` values
